.rtlcssrc

```diff
@@ -1,10 +1,6 @@
 {
-    "options": {
-        "preserveComments": true,
-        "preserveDirectives": false,
-        "swapLeftRightInUrl": true,
-        "swapLtrRtlInUrl": true,
-        "swapWestEastInUrl": false,
-        "autoRename": false
-    }
+	"options": {
+		"processUrls": { "atrule": true, "decl": false }
```

Comment:
What type of node types is this targeting? I couldn't quite grok what the doc was describing https://github.com/MohammadYounes/rtlcss#options-object